documentation/src/main/markdown/packagemgrs/conan.md

```diff
@@ -6,28 +6,28 @@
 
 ## Overview
 
-[solution_name] has two detectors for Conan:
+[solution_name] has two strategies for identifying components in Conan projects:
 
-* Conan Lockfile detector
-* Conan CLI detector
+* Conan CLI output parsing
+* Conan Lockfile parsing
 
 ## Conan detector requirements
 
 [solution_name] will run a Conan Detector if either of the following is true:
 
-* [solution_name] finds or is provided (via the *detect.conan.lockfile.path* property) a Conan lockfile. (If no lockfile is provided, [solution_name] looks for a file named conan.lock.) In this case, the Conan Lockfile detector runs and discovers dependency details using the contents of the Conan lockfile. The Conan Lockfile detector should always be preferred due to the additional information (package revisions) that may be provided by lockfiles.
-* [solution_name] finds a file named *conanfile.txt* or *conanfile.py*. In this case, the Conan CLI detector runs and discovers dependency details by running the *conan info* command on the Conan project and parsing the output.
+* [solution_name] finds or is provided via the *detect.conan.lockfile.path* property, a Conan lockfile. If no lockfile is provided, [solution_name] looks for a file named *conan.lock*. In this case, the Conan Lockfile detector runs and discovers dependency details using the contents of the Conan lockfile. For Conan version 1.x, the Conan Lockfile detector is preferred due to the additional information (package revisions) that may be provided by lockfiles. For Conan version 2.x, Conan CLI detector is preferred as a Conan 2.x lockfile is a flat list of all components rather than a graph.
```

Comment:
Should we not say "Conan CLI detector is **used**" instead of "... **preferred**"?

Reply:
This wording was like this before and I suspect it's to underline that that's what will be attempted first but there are other options.

Perhaps "detector will be attempted first" is the way to go?